autoware_plugin/src/autoware_plugin.cpp

```diff
@@ -48,12 +55,45 @@ namespace autoware_plugin
         pnh_->param<double>("trajectory_point_spacing", trajectory_point_spacing_, 0.1);
     }
 
+
     void AutowarePlugin::run()
     {
         initialize();
         ros::CARMANodeHandle::spin();
     }
 
+
+
+    bool AutowarePlugin::plan_trajectory_cb(cav_srvs::PlanTrajectoryRequest &req, cav_srvs::PlanTrajectoryResponse &resp){
+        resp.trajectory_plan = trajectory_msg;
+        resp.related_maneuvers.push_back(0);
+        resp.maneuver_status.push_back(1);
```

Comment:
Us the maneuver status enum here instead of a raw int value
cav_srvs::PlanTrajectory::MANEUVER_IN_PROGRESS

Reply:
modified accordingly